docs/tutorials/t-10.md

```diff
@@ -0,0 +1,118 @@
+# Algo Builder Web Support for Logic Signature Transaction
+
+The purpose of this tutorial is to demonstrate the usage of Algo Builder Web Package to sign and send a Logic Signature Transaction using major Algorand wallets (Algo Signer, Wallet Connect, MyAlgoWallet).
+
+### Requirements
+
+- [Algo Builder](https://algobuilder.dev/) and it's [dependencies](https://algobuilder.dev/guide/README#requirements)
+- Setup the accounts on [`Algosigner`](https://github.com/PureStake/algosigner), [`Wallet Connect`](https://walletconnect.com/) wallet and [`MyAlgo Wallet`](https://wallet.myalgo.com/). It could be MainNet, TestNet or private net.
+
+## Steps
+
+In this tutorial we will use Algo Builder Wallet Widget to connect to various web wallets. Then we will show how to instrument the wallet to sign a multisignature transaction. See [Wallet Widget tutorial](https://developer.algorand.org/solutions/vuejs-multi-wallet-widget/) for more details about the widget.
```

Comment:
wrong copy paste. Please update this section accordingly.

Reply:
oh yes, will update